resdk/scripts.py

```diff
@@ -350,3 +352,85 @@ def upload_reads():
         else:
             print("\nERROR: Incorrect file path(s).\n")
             exit(1)
+
+
+def update_knowledge_base():
+    """Updates the knowledge base from an external file."""
+    description = """Updates the remote knowledge base from an external file."""
+
+    parser = argparse.ArgumentParser(formatter_class=argparse.RawTextHelpFormatter,
+                                     description=description)
+
+    parser.add_argument('-a', '--address', default='https://torta.bcm.genialis.com',
+                        help='Resolwe server address')
+    parser.add_argument('-e', '--email', default='admin', help='User name')
+    parser.add_argument('-p', '--password', default='admin', help='User password')
+    parser.add_argument('-v', '--verbose', action='store_true', help='Enable verbose reporting')
+    parser.add_argument('features', help='ZIP file containing feature descriptions')
+
+    args = parser.parse_args()
+
+    if args.verbose:
+        resdk_logger.start_logging()
+
+    resolwe = Resolwe(args.email, args.password, args.address)
+
+    subtype_map = {
```

Comment:
@jkokosar this should be a mapping between what is found in the files being imported and the [internal designations](https://github.com/genialis/resolwe-bio/pull/107/files#diff-d65dd8283ab5cdf96b0e933ba90a5bcfR15). Everything, which is not in this list will be imported as `other`, so can you check how these types should map and update this dictionary?